pkg/ingress/controller/controller.go

```diff
@@ -579,15 +589,26 @@ func (c *Controller) deleteIngress(ing *nwv1.Ingress) error {
 		return nil
 	}
 
-	// Delete the floating IP for the load balancer VIP. We don't check if the Ingress is internal or not, just delete
-	// any floating IPs associated with the load balancer VIP port.
-	logger.Debug("deleting floating IP")
-
-	if _, err = c.osClient.EnsureFloatingIP(true, loadbalancer.VipPortID, "", ""); err != nil {
-		return fmt.Errorf("failed to delete floating IP: %v", err)
+	// Manage the floatingIP
+	keepFloatingSetting := getStringFromIngressAnnotation(ing, IngressAnnotationLoadBalancerKeepFloatingIP, "false")
+	keepFloating, err := strconv.ParseBool(keepFloatingSetting)
+	if err != nil {
+		return fmt.Errorf("unknown annotation %s: %v", IngressAnnotationLoadBalancerKeepFloatingIP, err)
 	}
 
-	logger.WithFields(log.Fields{"lbID": loadbalancer.ID}).Info("VIP or floating IP deleted")
+	if keepFloating {
+		logger.Debug("keep floating IP")
```

Comment:
I'm sorry, I don't understand your question. The idea here is, by default, when you delete your ingress, the Octavia LB and attached FIP are deleted.

Reply:
In the log message ? On openstack side the fip had a comment with ingress name and cluster name and this comment is kept as is, needed to be updated ?

Reply:
My point is that this looks like a leftover log from debugging. If it's not and you want to keep it I'm fine, but it has to include more info (which Ingress and FIP is affected) and maybe the reason (because Ingress has keep-IP annotation).

Reply:
+1 

given we have chance to update this code, better to enhance this 
current log seems useless